queue_services/entity-filer/tests/unit/filing_processors/test_annual_report.py

```diff
@@ -27,23 +28,43 @@
 from tests.unit import (
     create_business,
     create_filing,
+    factory_batch,
+    factory_batch_processing
 )
+from tests import EPOCH_DATETIME
 
 
-def test_process_ar_filing(app, session):
+@pytest.mark.parametrize('test_name,flag_on,in_dissolution,eligibility,legal_type,', [
+    ('AR successfully', True, False, False, 'CP'),
+    ('AR successfully', True, False, False, 'BC'),
+    ('Not withdrawn from the dissolution process', True, True, True, 'BC'),
+    ('Withdrawn from the dissolution process', True, True, False, 'BC'),
+    ('AR successfully when flag is off', False, True, False, 'BC')
```

Comment:
Can we add one last test for CPs?  

`(('AR successfully when flag is off', False, False, False, 'CP'))`

Reply:
Added